partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -52,8 +52,37 @@ statement::[
   query::{
     root: rex,
   },
+  d_d_l::{
+    op: ddl_op,
+  },
```

Comment:
This is more of a nit, but the generated code would be quite nice to keep the cascading heirarchy of: `DDL.Op`. AKA:
```suggestion
  d_d_l::{
    op: op::[
      create_table::{ ... },
      ...
    ]
  },
```

Or, if `op` is always going to be the only property, then:
```suggestion
  define::[
    create_table::{ ... },
    ...
  ],
```

Reply:
In the feature, I hope that the every DDL node will carry the shape of the resulting table for ease of processing. 

This is not needed now as the only operation we support is `CREATE TABLE`, but it could be helpful when we want to support ALTER, etc.